python-sdk/src/astro/databases/snowflake.py

```diff
@@ -663,7 +665,7 @@ def load_pandas_dataframe_to_table(
 
     def get_sqlalchemy_template_table_identifier_and_parameter(
```

Comment:
Change this to static?

Reply:
isn't this is also an override of the base class method?